node/src/main/scala/coop/rchain/node/node.scala

```diff
@@ -59,19 +56,20 @@ class NodeRuntime(conf: Conf)(implicit scheduler: Scheduler) {
   implicit val timeEffect: Time[Task]                     = effects.time
   implicit val metricsEffect: Metrics[Task]               = effects.metrics
   implicit val inMemoryPeerKeysEffect: KeysStore[Task]    = effects.remoteKeysKvs(remoteKeysPath)
+  val net                                                 = new UnicastNetwork(src)
   implicit val nodeDiscoveryEffect: NodeDiscovery[Task]   = effects.nodeDiscovery[Task](net)
   implicit val transportLayerEffect: TransportLayer[Task] = effects.transportLayer[Task](net)
-
   implicit val casperEffect: MultiParentCasper[Effect] = MultiParentCasper.hashSetCasper[Effect](
-//  TODO: figure out actual validator identities...
+    //  TODO: figure out actual validator identities...
     com.google.protobuf.ByteString.copyFrom(Array((scala.util.Random.nextInt(10) + 1).toByte)),
     storagePath,
     storageSize
   )
-
   implicit val packetHandlerEffect: PacketHandler[Effect] = effects.packetHandler[Effect](
     casperPacketHandler[Effect]
   )
+  implicit val protocolDispatcher: ProtocolDispatcher[Effect, SocketAddress] =
+    effects.RChainProtocolDispatcher.get[Effect]
```

Comment:
Maybe you can move these implicits to the top to the others?

Reply:
Will be removed in next PR